Tasks/Sergey Yashchanka/Task_2/returning_time.py

```diff
@@ -0,0 +1,117 @@
+# Задание:
+
+# Реализовать:
+# 1
+# .текстовый вывод текущего времени
+# 2
+# .текстовый вывод времени, введённого с консоли (hh :mm).
+# (дать пользователю возможность выбрать режим работы программы)
+#
+# Для получения текущего времени использовать модуль datetime.
+#
+# min == 0: тако е -то значение часа ровно
+# min < 30: стольк о -то минут следующего часа
+# min == 30: половина таког о -то
+# min > 30 and min < 45 стольк о -то минут следующего часа
+# min >= 45 без min таког о -то
+#
+#
+# 21 :16
+# шестнадцать минут десятого
+
+# ------------------------------------------------------------------------------------------------------------------ #
+
+# Решение:
+
+# from module import class
+from datetime import datetime as dt
+
+# get hours and minutes, translate format of time
+got_time = dt.now().time()
+got_hour = None
+got_min = None
+
+# user interaction
+answer_user = input(
+    "Здравствуйте! Если вы хотите получить текущее время введите: 'Y' либо введите время в формате 'hh:mm' | "
+)
+
+# check enter of user!! if enter is not correct, user get present time!
+try:
+    if 0 < int(answer_user[:answer_user.find(":")]) < 25 and -1 < int(
+            answer_user[answer_user.find(":") + 1:]) < 60:
+        got_hour = int(answer_user[:answer_user.find(":")]) % 12
+        got_min = int(answer_user[answer_user.find(":") + 1:])
+except ValueError:
+    got_hour = got_time.hour % 12
+    got_min = got_time.minute
```

Comment:
a piece of advice here - check for the general Error as well, just in case, and notify user about a mistake explicitly